libretro_core_options.h

```diff
@@ -706,12 +706,12 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(memcard_right_index),
       "Memory Card Right Index",
       NULL,
-      "Changes the memory card currently loaded in the right slot. This option will only work if Memory Card 1 is enabled. The default card is index 1.",
+      "Change the memory card currently loaded in the right slot. This option will only work if Memory Card 1 is enabled. The default card is index 1.",
```

Comment:
Lowercase the second "Memory Card" to unify all cases. Seems that Sony didn't trademark the words "Memory Card", so they don't require mandatory uppercasing.